src/pages/Events/index.tsx

```diff
@@ -90,8 +95,34 @@ const Events = () => {
     event('search', 'events', JSON.stringify(data));
     setFilters(data);
     navigate(`${location.pathname}${argsToParams(data)}`, { replace: true });
+    !lgDown || setExpanded((prev) => !prev);
   };
 
+  const [expanded, setExpanded] = useState(false);
```

Comment:
Denne kan ha et mer beskrivende navn, slik som for eks `searchFormExpanded, setSearchFormExpanded`